pkg/instrumentation/podmutator.go

```diff
@@ -19,9 +20,55 @@ import (
 	"github.com/aws/amazon-cloudwatch-agent-operator/internal/webhookhandler"
 )
 
+const (
+	defaultExporterEndpoint                = "http://amazon-cloudwatch-agent.amazon-cloudwatch:4317"
+	defaultJavaImage                       = "160148376629.dkr.ecr.us-west-2.amazonaws.com/aws-apm-preview:latest"
+	defaultAPIVersion                      = "cloudwatch.aws.amazon.com/v1alpha1"
+	defaultInstrumenation                  = "java-instrumentation"
+	defaultNamespace                       = "default"
+	defaultKind                            = "Instrumentation"
+	otelSampleEnabledKey                   = "OTEL_SMP_ENABLED"
+	otelSampleEnabledDefaultValue          = "true"
+	otelTracesSamplerArgKey                = "OTEL_TRACES_SAMPLER_ARG"
+	otelTracesSamplerArgDefaultValue       = "0.05"
+	otelTracesSamplerKey                   = "OTEL_TRACES_SAMPLER"
+	otelTracesSamplerDefaultValue          = "parentbased_traceidratio"
+	otelExporterTracesEndpointKey          = "OTEL_EXPORTER_OTLP_TRACES_ENDPOINT"
+	otelExporterTracesEndpointDefaultValue = "http://amazon-cloudwatch-agent.amazon-cloudwatch:4317"
```

Comment:
i guess we don't need this one since we will always have metrics/traces go through the same endpoint